### ISSUE TITLE
Huge Close, Minimize, Maximize buttons on Budgie

### ISSUE THREAD

**Issue by matricea** [adapta-project/adapta-gtk-theme#254]
**Reporter info**

 ```
 * Distribution - Solus 1.2.1
 * Gtk+ 3.0 version - 3.20.9
 * Desktop environment - Budgie 10.2.8
 ```

**Adapta version (if you knew)**

3.22.1.134-24

**Related Application and/or Shell theming (or name of widget)**

Nautilus, many GNOME apps

**Actual issue**

I'm using Solus 1.2.1 with the Budgie desktop and for some reason the Close, Minimize and Maximize buttons appear huge on some windows, such as Nautilus, but smaller on other, such as Firefox or GNOME Terminal. I've also used Adapta on Unity with Ubuntu 16.10 and all buttons are the same size there. Is there any way to alter the settings?

**Steps to reproduce (if you knew)**

![screenshot from 2016-10-20 19-00-06](https://cloud.githubusercontent.com/assets/22963097/19568034/aa974902-96f8-11e6-933d-fd601ab9fd7f.jpg)

**Expected behaviour**

Smaller icons.

**Other Note (feature-request, question, etc...)**

feature-request: A tool to configure all aspects of Adapta.

**Comment by tista500** [comment]
o_O??

What are you talking about? Was this report is theme specific?
If you meant that both Server-side-decoration and Client-side-decoration should be the same height of GtkHeaderBar, it can't be. There's no theme improves such thing. See Adwaita. GtkButtons should be optimized by the geometry of its parent GtkWidgets, in this case, GtkHeaderBar or Mutter's titlebar.

> I've also used Adapta on Unity with Ubuntu 16.10 and all buttons are the same size there

Unity7 uses Compiz but Budgie-desktop does Mutter.

**Comment by matricea** [comment]
Hey Tista! I'm using Solus, and the default Arc theme has smaller buttons everywhere, but I like Adapta more... so if it's Mutter's fault, how should I proceed? Thanks!

**Comment by caoli5288** [comment]
I think the bigger icons are aesthetically pleasing. . . This is a matter of personal aesthetic, not themes

**Comment by ikeydoherty** [comment]
TL;DR Small buttons will be found on fixed-size server-side decorations (i.e. Firefox etc, *non* CSD) and chunky monkies will be found on everything with a GtkHeaderBar, because apparently it's cool to use this widget everywhere for everything :p This isn't a mutter issue anyway.

**Comment by tista500** [comment]
@matricea,

> the default Arc theme has smaller buttons everywhere

Adapta is not the fork of Arc.
So please thell me what is the benefit to shrink its 'clickable-area' by using tiny button images inside the GtkHeaderBar (i.e. 20px x 20px buttons inside the 42px height of GtkHeaderBar).

@caoli5288,

 > I think the bigger icons are aesthetically pleasing

Agreed.

@ikeydoherty,

> This isn't a mutter issue anyway.

That's right and theme-side instead (especially `.default-decoration` style-class).

Regards.

**Comment by TheAifam5** [comment]
Would be nice to have small buttons inside of Headerbar :)

**Comment by matricea** [comment]
Hey Tista, the bigger icons are horrible from my point of view, they ruin the beauty of the interface. It's all about aesthetics I guess, especially when I've migrated from an OS where I've used Adapta for several months and, by default, it offered smaller button images on the header bar.

I don't understand why Adapta can't look the same on another desktop environment (sorry, I don't have any GTK+ theming knowledge). Also, why is the Eta version of Adapta and Adapta-Nokto using a bit smaller buttons on Budgie? So, again, can you please point me into the right direction of the file I need to edit to customize my header bar?

Thanks!

**Comment by tista500** [comment]
> the bigger icons are horrible from my point of view

Why you use my theme? Feel free to go other theme.

> It's all about aesthetics I guess

Yes. And no evidence in your post.

> why Adapta can't look the same on another desktop environment

Gnome-Shell, Budgie, XFce4, and Mate, most of DEs have the similar interface design except for Unity7.

> why is the Eta version of Adapta and Adapta-Nokto using a bit smaller buttons on Budgie? 

Read the README first.

https://github.com/adapta-project/adapta-gtk-theme/blob/master/gtk/sass/3.20/_common.scss

Regards.
